stingray/spectroscopy.py

```diff
@@ -331,7 +351,7 @@ def get_mean_phase_difference(cs, model):
     return avg_psi, stddev
 
 
-def get_phase_lag(cs, model):
+def get_phase_lag(cs: Crossspectrum, model: Any) -> Tuple[float, float, float]:
```

Comment:
Can use `astropy.modeling.core.Model` instead of any for the model as its the base class of all the astropy models